packages/filebrowser/src/browser.ts

```diff
@@ -200,8 +205,8 @@ export class FileBrowser extends Widget {
   /**
    * Download the currently selected item(s).
    */
-  download(): void {
-    this._listing.download();
+  download(): Promise<void> {
```

Comment:
Another API change: allow users to hook into when the download promise is resolved.